openshift/e2e-common.sh

```diff
@@ -204,6 +204,24 @@ install_serverless_operator_branch() {
   unset OPENSHIFT_CI
   ./hack/install.sh || failed=1
   subheader "Successfully installed serverless operator."
+
+  subheader "Install Kafka from serverless-operator repo"
+  INSTALL_SERVING="false" INSTALL_KAFKA="true" ./hack/install.sh || failed=1
+  subheader "Successfully installed kafka."
+
+  header "Applying Strimzi Topic CR"
+  cat <<-EOF | oc apply -f - || failed=1
```

Comment:
```suggestion
  cat <<-EOF | oc apply -n kafka -f - || failed=1
```
@dsimansk : I think we'd need to mention the namespace here.

Reply:
Ahh, that's a great point. I'm about to test it on CRC, I'll try to add namespace as well.